modules/programs/aerc-accounts.nix

```diff
@@ -0,0 +1,131 @@
+{ config, lib, pkgs, confSections, confSection, ... }:
+
+with lib;
+
+let
+  mapAttrNames = f: attr:
+    with builtins;
+    listToAttrs (attrValues (mapAttrs (k: v: {
+      name = f k;
+      value = v;
+    }) attr));
+  addAccountName = name: k: "${k}:account=${name}";
+in {
+  type = mkOption {
+    type = types.attrsOf (types.submodule {
+      options.aerc = {
+        enable = mkEnableOption "aerc";
+        extraAccounts = mkOption {
```

Comment:
Please separate option definitions by empty line.

Reply:
Resolved in #3150